mission-control/docs/config-db/concepts/changes.md

```diff
@@ -0,0 +1,81 @@
+# Changes
+
+Once a config item has been saved to ConfigDB, it's changes are also tracked. The changes could come from external sources like Kubernetes events, Azure activities, ... Or the changes can be auto detected by comparing the old config with the newly changed config.
```

Comment:
Split this up into the different types of changes (Diff vs Event) and how they are created and what they relate to